server/src/main/java/org/jetbrains/bsp/bazel/server/bloop/BloopExporterInitializer.java

```diff
@@ -0,0 +1,53 @@
+package org.jetbrains.bsp.bazel.server.bloop;
+
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier;
+import io.vavr.control.Option;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import org.jetbrains.bsp.bazel.server.bsp.info.BspInfo;
+import org.jetbrains.bsp.bazel.workspacecontext.DefaultWorkspaceContextProvider;
+
+public class BloopExporterInitializer {
+  public static void main(String[] args) {
+    Option<Path> projectViewPath;
+    Path bspProjectRoot = Paths.get("").toAbsolutePath().getParent();
+    Path projectRoot;
+
+    if (args.length == 1) {
+      projectRoot = Paths.get(args[0]);
```

Comment:
it's not a project root - it's a pwd of the installer

Reply:
in this case its used as the project root.